sys/linux/devio.txt

```diff
@@ -0,0 +1,221 @@
+# Copyright 2019 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+# https://elixir.bootlin.com/linux/latest/source/drivers/usb/core/devio.c
+
+include <linux/ioctl.h>
+include <linux/usb.h>
+include <linux/usbdevice_fs.h>
+include <uapi/linux/fcntl.h>
+include <uapi/linux/usb/ch9.h>
+
+define MAXDRIVERNAME	USBDEVFS_MAXDRIVERNAME + 1
+
+resource fd_usbfs[fd]
+
+syz_open_dev$usbfs(dev ptr[in, string["/dev/bus/usb/00#/00#"]], id intptr, flags flags[open_flags]) fd_usbfs
+
+read$usbfs(fd fd_usbfs, buf buffer[out], count len[buf])
+
+mmap$usbfs(addr vma, len len[addr], prot flags[mmap_prot], flags flags[mmap_flags], fd fd_usbfs, offset fileoff)
+_ = __NR_mmap2
+
+# TODO: arg is a pointer to an URB.
+ioctl$USBDEVFS_REAPURB(fd fd_usbfs, cmd const[USBDEVFS_REAPURB], arg ptr[out, int64])
+ioctl$USBDEVFS_REAPURBNDELAY(fd fd_usbfs, cmd const[USBDEVFS_REAPURBNDELAY], arg ptr[out, int64])
+
```

Comment:
Any reason for this whitespace?

Reply:
I wanted to emphasize that the comment is for `ioctl$USBDEVFS_REAPURB` and also for `ioctl$USBDEVFS_REAPURBNDELAY`. Should I remove it?

Reply:
Ah, OK, no, you can leave it.